tests/unit/modules/win_firewall_test.py

```diff
@@ -28,7 +28,7 @@
 IS_WIN = win_firewall.__virtual__()
 
 
-@skipIf(not IS_WIN, "This test case runs only on Windows system")
+@skipIf(not IS_WIN[0], "This test case runs only on Windows system")
```

Comment:
That makes sense. But currently this skipIf does nothing on non-windows systems. So I think either I need to change the way we check for windows in the test or just remove the skipIf all together. What do you think? I have the tests passing on linux with my other PR, so the skip may actually not even be necessary.